content/docs/user-guide/testing/parallel-pattern/_index.md

```diff
@@ -0,0 +1,526 @@
+---
+description: ' Use parallel pattern to allow tests to be run in parallel. '
+title: Parallel Pattern & Batching Test-Writing
+---
+
```

Comment:
I would suggest an introductory section that explains the difference of parallel/batching tests and why the user would want to use this